apps/table-frontend/src/app/components/effective.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react';
+import { getFile } from './get-file-info';
+import {
+  ParsedArticleParagraphStandalone,
+  ArticleParagraphBacksUpClaim,
+} from '@foodmedicine/interfaces';
+export interface EffectiveProps {
```

Comment:
can you leave a comment explaining what EffectiveProps means?

Reply:
Ok sure, will do!